src/CommonLib/resources.h

```diff
@@ -28,8 +28,9 @@
 #define ASPNETCORE_EVENT_INPROCESS_THREAD_EXIT_STDERR_MSG    L"Application '%s' with physical root '%s' hit unexpected managed background thread exit, ErrorCode = '0x%x. First 4KB characters of captured stderr logs on startup:\r\n%s"
 #define ASPNETCORE_EVENT_INPROCESS_THREAD_EXIT_STDOUT_MSG    L"Application '%s' with physical root '%s' hit unexpected managed background thread exit, ErrorCode = '0x%x. Last 4KB characters of captured stdout and stderr logs:\r\n%s"
 #define ASPNETCORE_EVENT_INPROCESS_THREAD_EXIT_MSG           L"Application '%s' with physical root '%s' hit unexpected managed background thread exit, ErrorCode = '0x%x. Please check the stderr logs for more information."
-#define ASPNETCORE_EVENT_RECYCLE_APPOFFLINE_MSG              L"Application '%s' is recycled due to app_offline file was detected."
-#define ASPNETCORE_EVENT_RECYCLE_CONFIGURATION_MSG           L"Application '%s' is recycled due to configuration changed"
+#define ASPNETCORE_EVENT_RECYCLE_APPOFFLINE_MSG              L"Application '%s' is recycled due to app_offline file."
+#define ASPNETCORE_EVENT_RECYCLE_CONFIGURATION_MSG           L"Application '%s' is recycled due to configuration change"
+#define ASPNETCORE_EVENT_RECYCLE_FAILURE_CONFIGURATION_MSG   L"Failed to recyle application for configuration change and had to recycle IIS worker process, Error Code = '0x%x'."
```

Comment:
nit: "Failed to recycle application due to configuration change and had to recycle IIS worker process."